tasks/post_copy_request_to_queue/test/unit_tests/test_post_copy_request_to_queue.py

```diff
@@ -782,19 +797,3 @@ def test_query_db_exceptions_bubble_up(
                 "status_id": shared_recovery.OrcaStatus.PENDING.value
             }
         )
-
-    def test_get_metadata_sql_happy_path(self):
```

Comment:
I think we still need this happy path test.

Reply:
We don't typically check raw sql because it isn't a functional check. I can reintroduce this if we want, but it'll only validate that the sql was copy-pasted from the underlying function.

Reply:
I see. In that case I think it is okay to remove the test.